File: app/src/main/java/org/dhis2hiv/usescases/datasets/datasetDetail/DataSetDetailComponent.java
package org.dhis2hiv.usescases.datasets.datasetDetail;

import org.dhis2hiv.data.dagger.PerActivity;

import dagger.Subcomponent;


@Subcomponent (modules = DataSetDetailModule.class)
@PerActivity
public interface DataSetDetailComponent {
    void inject(DataSetDetailActivity activity);
}

File: app/src/main/java/org/dhis2hiv/usescases/datasets/dataSetTable/dataSetDetail/DataSetDetailComponent.kt
package org.dhis2hiv.usescases.datasets.dataSetTable.dataSetDetail

import dagger.Subcomponent
import org.dhis2hiv.data.dagger.PerFragment

@Subcomponent(modules = [DataSetDetailModule::class])
@PerFragment
interface DataSetDetailComponent {
    fun inject(dataSetDetailFragment: DataSetDetailFragment)
}
